### PR TITLE
[NIC-41] Add complete date and person info to document

### PR DESCRIPTION
**Pull request by AksyonovaKaterina**

### CODE REVIEW

File: backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt [hunk 1]
```diff
@@ -49,7 +53,11 @@ class DefaultDocumentComponent(
             .awaitSingle()
     }
 
-    override suspend fun getAll(types: List<DocumentType>, statuses: List<DocumentStatus>, pageable: Pageable): List<DocumentEntity> {
+    override suspend fun getAll(
+        types: List<DocumentType>,
+        statuses: List<DocumentStatus>,
+        pageable: Pageable
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt]: comma 🤡

File: backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt [hunk 2]
```diff
@@ -49,7 +53,11 @@ class DefaultDocumentComponent(
...
+    ): List<PersonDocumentEntity> {
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt:59]: maybe use `PersonDocumentEntity` also to get a single object (the method above)?

and for `update` method?

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/ContactsDto.kt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.nosql.document.dto
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import javax.validation.constraints.Email
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+class ContactsDto (
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/dto/ContactsDto.kt:10]: use `data class`

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDocumentResponseDto.kt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package com.nosql.document.dto
+
+class PersonDocumentResponseDto(
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDocumentResponseDto.kt:3]: use `data class`

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDto.kt [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package com.nosql.document.dto
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+class PersonDto(
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDto.kt:5]: use `data class`

File: backend/document-service/src/main/kotlin/com/nosql/document/entity/ConfidentialEntity.kt [hunk 6]
```diff
@@ -0,0 +1,22 @@
+package com.nosql.document.entity
+
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.mapping.Field
+
+class ConfidentialEntity(
```
Comment (V7074) [inline backend/document-service/src/main/kotlin/com/nosql/document/entity/ConfidentialEntity.kt]: I think you should not use this `entity` in `document-service`

**Review by V7074 — CHANGES_REQUESTED** [review]

File: backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt (hunk 2 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt:59]: make in update

File: backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt (hunk 1 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/component/impl/DefaultDocumentComponent.kt]: fixed

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/ContactsDto.kt (hunk 3 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/dto/ContactsDto.kt:10]: this make no sense, we do not use toString

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDocumentResponseDto.kt (hunk 4 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDocumentResponseDto.kt:3]: this make no sense, we do not use toString

File: backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDto.kt (hunk 5 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/dto/PersonDto.kt:5]: this make no sense, we do not use toString

File: backend/document-service/src/main/kotlin/com/nosql/document/entity/ConfidentialEntity.kt (hunk 6 above)
Comment (AksyonovaKaterina) [reply backend/document-service/src/main/kotlin/com/nosql/document/entity/ConfidentialEntity.kt]: fixed

**Review by V7074 — APPROVED** [review]

**Pull request merged**